src/ui/display.h

```diff
@@ -15,6 +15,13 @@
 #include "../common/script.h"
 #include "../constants.h"
 
+#define MESSAGE_CHUNK_SIZE        64
+#define MESSAGE_CHUNK_PER_DISPLAY 2
+#define MESSAGE_MAX_DISPLAY_SIZE \
+    (MESSAGE_CHUNK_SIZE * MESSAGE_CHUNK_PER_DISPLAY + 2 * sizeof("...") - 1)
+#define MAX_DISPLAYBLE_CHUNK_NUMBER \
+    (5 * MESSAGE_CHUNK_PER_DISPLAY)  // If the message is too long we will not display it
```

Comment:
I think `MAX_DISPLAYBLE_CHUNK_NUMBER` should be defined in `sign_message.c`, as it's a UX decision of the signing flow, rather than being needed to allocate the ui structs.